scripts/agent-dev

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
-set -e
```

Comment:
We should also update [DEVELOPING.md](https://github.com/rancher/fleet/blob/master/DEVELOPING.md) which still has references to `make agent-dev`.